ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -23,7 +23,8 @@ static_library('nnstreamer_decoder_direct_video',
 
 # image labeling
 decoder_sub_image_labeling_sources = [
-  'tensordec-imagelabel.c'
+  'tensordec-imagelabel.c',
+  'tensordecutil.c'
```

Comment:
How about making this src file be a common dep for all decoder sub-plugins?

Reply:
Image-segment and Directvideo don't use it yet.